Gopkg.toml

```diff
@@ -42,12 +42,29 @@ required = [
 
 [[constraint]]
   name="sigs.k8s.io/controller-runtime"
-  version="v0.1.10"
+  version="v0.2.0-beta.5"
 
 [[override]]
   name="sigs.k8s.io/controller-tools"
-  source = "https://github.com/crossplaneio/controller-tools.git"
-  revision = "9acc926ef7ce7372609b34fbafe397158e6442ef"
+  source = "https://github.com/muvaf/controller-tools.git"
+  revision = "301a879da11105676f319f9080278a54b956d07f"
+
+# temp stanzas
```

Comment:
we should take another pass through these dependency rules to see if any can be removed.  I have a feeling that some were needed while we were still getting through the migration, but they might not all be needed at a steady state.

https://golang.github.io/dep/docs/Gopkg.toml.html

Reply:
Is there a reason we are keeping this file?
If you run `go mod init github.com/crossplaneio/crossplane` in this folder it automatically converts `Gopkg.toml` to `go.mod`, and you can get rid of this file.

Reply:
@soorena776 while I support the movement to go modules, I think that should be addressed in a separate dedicated PR.